yt_dlp/extractor/tnaflix.py

```diff
@@ -92,9 +92,8 @@ def _real_extract(self, url):
         if not cfg_url:
             inputs = self._hidden_inputs(webpage)
             if inputs.get('vkey') and inputs.get('nkey'):
-                cfg_url = f'https://www.{host}.com/cdn/cdn.php'
+                cfg_url = f"https://cdn-fck.{host}.com/{host}/{inputs['vkey']}.fid"
```

Comment:
change `cfg_url` to `http` here to avoid cert problems
single-quotes for outer and double-quotes for inner
```suggestion
                cfg_url = f'http://cdn-fck.{host}.com/{host}/{inputs["vkey"]}.fid'
```